src/components/download-item/download-item.tsx

```diff
@@ -46,13 +47,16 @@ export const DownloadItem = withText({
       downloadUrl,
       iconFileType,
       isDefault,
-      player
+      player,
+      displayFirst
     }: DownloadItemProps) => {
       const downloadItemRef = useRef<HTMLDivElement>(null);
 
       useEffect(() => {
         if (isDefault) {
```

Comment:
why not if (isDefault || displayFirst) ?

Reply:
changed